package-lock.json

```diff
@@ -217,6 +217,12 @@
       "integrity": "sha512-dInnr7nSPsofnLggOf70xvsInUjf3tRrE8XmxsioXALWQHkwEWi7RhTBCa9mYmiUDHMtuanSDN/JOW187ChIhw==",
       "dev": true
     },
+    "@types/eslint-visitor-keys": {
```

Comment:
Oh wow. Actually these changes to package-lock.json are not at all what I was expecting. I was just expecting changes to fs-events and nothing else.

I think there might be an issue with how Oppia is set up on your machine, I don't think anyone else has run into this. I'm not sure it makes sense to commit this package-lock.json. /cc @oppia/dev-workflow-team  @NishealJ any thoughts? (Please see #6914 too.)

Reply:
I also run into this @seanlip , I think this is a Mac problem since @sophiewu6 and I share the same OS.